From: Adrian Knoth <adi@drcomp.erfurt.thur.de>
Description: Fix FTBFS on hurd
 As of 2010-08-01, SA_SIGINFO isn't defined on hurd, so check before 
 setting it.
--- a/dbus/sigsegv.c
+++ b/dbus/sigsegv.c
@@ -177,7 +177,9 @@
 
     memset(&action, 0, sizeof(action));
     action.sa_sigaction = signal_segv;
+#ifdef SA_SIGINFO
     action.sa_flags = SA_SIGINFO;
+#endif
     if(sigaction(SIGSEGV, &action, NULL) < 0) {
         jack_error("sigaction failed. errno is %d (%s)", errno, strerror(errno));
         return 0;
